torchmetrics/functional/regression/symmetric_mean_absolute_percentage_error.py

```diff
@@ -0,0 +1,76 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Tuple
+
+import torch
+from torch import Tensor
+
+from torchmetrics.utilities.checks import _check_same_shape
+
+
+def _symmetric_mean_absolute_percentage_error_update(
+    preds: Tensor,
+    target: Tensor,
+    epsilon: float = 1.17e-06,
+) -> Tuple[Tensor, int]:
+
+    _check_same_shape(preds, target)
+
+    abs_diff = torch.abs(preds - target)
+    abs_per_error = abs_diff / torch.clamp(torch.abs(target) + torch.abs(preds), min=epsilon)
+
+    sum_abs_per_error = 2 * torch.sum(abs_per_error)
+
+    num_obs = target.numel()
+
+    return sum_abs_per_error, num_obs
+
+
+def _symmetric_mean_absolute_percentage_error_compute(sum_abs_per_error: Tensor, num_obs: int) -> Tensor:
+    return sum_abs_per_error / num_obs
+
+
+def symmetric_mean_absolute_percentage_error(preds: Tensor, target: Tensor) -> Tensor:
+    """
+    Computes symmetric mean absolute percentage error.
+
+    Args:
+        preds: estimated labels
+        target: ground truth labels
+
+    Return:
+        Tensor with SMAPE
+
+    Note:
```

Comment:
this note does not really make sense to the user without knowing how the epsilon influences the computation. I suggest removing it

Reply:
I removed it indeed. I originally kept it, in order to explain the default value used for epsilon, but it's true it does not make much sense since I'm not comparing it to a sklearn implementation